source/module_md/test/CMakeLists.txt

```diff
@@ -0,0 +1,342 @@
+remove_definitions(-D__MPI -D__LCAO )
+add_definitions(-D__CMD -D__NORMAL)
+
+AddTest(
+  TARGET md_LJ_pot
+  SOURCES LJ_pot_test.cpp
+  ../MD_func.cpp
```

Comment:
It would be better if you put the required sources into a list and reusing it in `AddTest` to eliminate duplicated code.